packages/emails/src/templates/index.ts

```diff
@@ -15,6 +15,7 @@ export { OrganizerCancelledEmail } from "./OrganizerCancelledEmail";
 export { OrganizerLocationChangeEmail } from "./OrganizerLocationChangeEmail";
 export { OrganizerPaymentRefundFailedEmail } from "./OrganizerPaymentRefundFailedEmail";
 export { OrganizerRequestEmail } from "./OrganizerRequestEmail";
+export { OrganizerRequestEmailV2 } from "./OrganizerRequestEmailV2";
```

Comment:
Moved to a new template so we can enable it on demand @anikdhabal

Reply:
Great workaround @zomars 🚀